compiler/semgnrc.nim

```diff
@@ -58,15 +58,22 @@ template isMixedIn(sym): bool =
 
 proc semGenericStmtSymbol(c: PContext, n: PNode, s: PSym,
                           ctx: var GenericCtx; flags: TSemGenericFlags,
-                          fromDotExpr=false): PNode =
+                          fromDotExpr: static bool = false): PNode =
```

Comment:
Why does this have to be  `static bool`? The code duplication is likely worse than a runtime check which is predicted correctly.

Reply:
It doesn't have to be, I thought since it's a small proc and called tons of times it might be better, but it's non-essential, changing it back